build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.6.3'
```

Comment:
For this issue, upgrading gradle version should not be done. If you want to upgrade gradle please send separate PR for that :)